app/mailers/devise_mailer.rb

```diff
@@ -5,5 +5,5 @@ class DeviseMailer < Devise::Mailer
   include Roadie::Rails::Automatic
   include Devise::Controllers::UrlHelpers
 
-  default from: 'Pupilfirst <noreply@pupilfirst.com>'
+  default from: "<#{Rails.application.secrets.default_sender_email_address}>"
```

Comment:
Shouldn't the `school_name` be used here as well (see `SchoolMailer`), to act as the sender's _name_?